prototypes/functions/HttpTriggerRedis/prototype-redis-cache-connect.py

```diff
@@ -0,0 +1,24 @@
+# Prototype of an Azure Function (running locally) that accesses 
+# Azure Redis cache [nsc-redis-dev-usw2-Thursday] and GETs/SETs data from 
+# that cache. (Run Redis locally using Docker)
+
+import logging
+import redis
+
+# Azure cache for Redis local access - DEVELOP
+REDIS_PORT = 6379
+REDIS_HOST = '127.0.0.1'
+cache = redis.Redis(host=REDIS_HOST, port=REDIS_PORT)
+
+# Azure cache for Redis access - PRODUCTION
+# REDIS_PORT = 6380
+# REDIS_HOST = 'nsc-redis-dev-usw2-thursday.redis.cache.windows.net'
+# REDIS_KEY = 'DQZfYhdrqhBVm5Mu3WgteoH0GihbxxRMZF6t15NlwNA='
+# cache = redis.StrictRedis(host=REDIS_HOST, port=REDIS_PORT, db=0, password=REDIS_KEY, ssl=True)
```

Comment:
I'm not familiar with Redis, but is it okay to have this key value in a public repo?

Reply:
I think in the future it needs to be replaced with a secret, and this key will need to be reset after use until then, but I will double check on that just in case. Since I'm not using it at all here it could just as easily be deleted

Reply:
Yeah you should delete this. I think once everyone knows how to use the environment variables, we'll recreate the redis server so it has a new key - this one has been compromised (by me and others all committing it at some point)